eras/conway/impl/CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # Version history for `cardano-ledger-conway`
 
-## 1.17.0.1
+## 1.17.1.0
```

Comment:
Not quite sure if this bump is correct

Reply:
Adding a new constructor to an existing type is a breaking change, so it needs to be `1.18.0.0`. Moreover, version in cabal file also needs to be bumped.